src/styles/blocks/card.scss

```diff
@@ -0,0 +1,106 @@
+:root {
+  --main-accent: #060b35;
+  --secondary: #616070;
+  --blue-accent: #00acdc;
+  --white: #fff;
+  --border-card-color: #f3f3f3;
+}
```

Comment:
If you are using a preprocessor you should use scss variables, and move them to variables.scss file